src/main/java/nl/tudelft/contextproject/model/level/Level.java

```diff
@@ -77,35 +88,28 @@ public MazeTile getTile(int x, int y) {
 	 * 		a JSONObject representing this Level
 	 */
 	public JSONObject toWebJSON() {
-		//Create a JSONObject
 		JSONObject json = new JSONObject();
-		
-		//Set the width and height
+
 		json.put("width", getWidth());
 		json.put("height", getHeight());
-		
-		//Store all the tiles in a JSONObject.
```

Comment:
I suggest readding this comment. This comment alone should suffice to explain the entire for loop below it.